deploy/helm/charts/values.yaml

```
@@ -144,6 +144,8 @@ lvmController:
     # Configure the maximum number of queries allowed after
     # accounting for rolled over qps from previous seconds.
     burst: 0
+  podAntiAffinity:
+    enable: false
```

For consistency let's use `enabled`

```suggestion
    enabled: false
```